guides/common/modules/proc_creating-compute-profiles.adoc

```diff
@@ -8,6 +8,12 @@ A default installation of {ProjectName} contains three predefined profiles:
 * `2-Medium`
 * `3-Large`
 
+You can apply the compute profiles to the following compute resource providers:
+
+* VMware
```

Comment:
Question:
Is there a difference between CRs supported in upstream and downstream?

Because complete list would include also the following:
* https://github.com/theforeman/foreman_google
* https://github.com/theforeman/foreman_azure_rm
* EC2 / AWS
* OpenStack

Reply:
I can only speak for orcharhino: orcharhino does not support OpenStack.

Reply:
@stejskalleos Is it the same list for applying Compute Profiles as it is for provisioning in general?

Reply:
Yes, every compute resource can have profiles, including bare metal which I forgot to list

Reply:
Really? How bare metal? Physical hosts have a given number of CPUs and memory, for what is a compute profile good there?

Reply:
Yeah you are right, no CPs for bare-metal.
Thinking where I got the idea that it's also for bare metals.
Sorry!